app/templates/components/widgets/forms/ticket-input.hbs

```diff
@@ -179,13 +179,15 @@
           <div class="three wide field">
             <label class="required text muted" for="ticket_min_order">{{t 'Minimum Order'}}</label>
             <Input
+oninput="this.value = Math.abs(this.value)"
```

Comment:
I don't believe this'll work

Reply:
The least number of tickets I can type manually is 0(even if I use the "-" sign) and same goes for the case when I use arrows of the number input. Although, the minimum number should be one which I just read and I have changed it accordingly, is there something else that I am missing here?